Content.Shared/CCVar/CCVars.cs

```diff
@@ -1202,6 +1202,9 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<bool> ChatShowTypingIndicator =
             CVarDef.Create("chat.show_typing_indicator", true, CVar.CLIENTONLY);
 
+        public static readonly CVarDef<string> MOTD =
+            CVarDef.Create("chat.motd", "", CVar.SERVER, "A message broadcasted to every player that joins the server.");
```

Comment:
Im like 90% sure the description var on Create() is useless, just move it to a xml comment above this

Reply:
Done